apps/coscrad-frontend/src/components/resources/bibliographic-references/shared/creators-presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IBibliographicReferenceCreator } from '@coscrad/api-interfaces';
+import { Box, Typography } from '@mui/material';
+import { CommaSeparatedList } from 'apps/coscrad-frontend/src/utils/generic-components/presenters/styled-comma-separated-list';
+import { CreatorPresenter } from './creator-presenter';
+
+interface CreatorsPresenterProps {
+    creators: IBibliographicReferenceCreator[];
+}
+
+export const CreatorsPresenter = ({ creators }: CreatorsPresenterProps): JSX.Element => {
+    {
+        /* TODO: create uppercase label for `type` */
+    }
+
+    const creatorLabel = creators.length > 1 ? `Creators` : `Creator`;
+
+    return (
+        <Box sx={{ mb: 1 }}>
+            <Typography component="span" sx={{ fontWeight: 'bold' }}>
+                {creatorLabel}:&nbsp;
+            </Typography>
+            <CommaSeparatedList>
+                {creators.map(({ name, type }, index) => (
+                    <span>
```

Comment:
This span could be within `CreatorPresenter`, but it does help to use `span` to give an inline list, which is likely the only use for comma separated list.

Reply:
Can't you wrap the spans inside of `CommaSeparatedList`? That's where the responsibility lies.